packages/core/src/subgraph/getUserStakedPools.ts

```diff
@@ -0,0 +1,79 @@
+import { PoolStake, SentryKey } from "@sentry/sentry-subgraph-client";
+import { GraphQLClient, gql } from 'graphql-request'
+import { config } from "../config.js";
+import { BulkOwnerOrPool } from "../operator/operatorRuntime.js";
+
+
+/**
+ * Find all pools specific wallets have at least one key staked in
+ * @param wallets - The filter for the wallets we want to get pools for
+ * @param excludePools - Filter for pools to exclude if we already loaded them
+ * @param includeSubmissions - If the submissions should be included
+ * @param submissionsFilter - The filter for the submissions if submissions should be included
+ * @returns List of sentry pool infos
+ */
+export async function getUserStakedPools(
+  wallets: string[],
```

Comment:
Possibly update name to `getUserStakedPoolsFromSubgraph`?

Reply:
Renamed to follow the naming pattern, thank you!